Gemfile

```diff
@@ -1,7 +1,9 @@
 source "https://rubygems.org"
 
 group :docker do
-  gem 'rake-tasks-docker', :git => 'https://github.com/inviqa/inviqa-rake-tasks-docker.git'
+  gem 'rake-tasks-docker', '~> 0.1.0'
+  gem 'rake-tasks-docker-setup', '~> 0.1.0'
+  gem 'rake-tasks-docker-sync', git: 'git@github.com:inviqa/inviqa-rake-tasks-docker-sync.git'
```

Comment:
I can't push this one to rubygems repo due to missing teams on the github repo. HELPDESK-6561 raised

Reply:
this will need updating when we push the updates to the gemss